core/src/RecordAuditParams.ts

```diff
@@ -0,0 +1,5 @@
+export class RecordAuditParams {
+    "oid": string
+    "Date From": Date
```

Comment:
We don't normally label variables this way. please use camel case (e.g. dateFrom)